kratos/tests/cpp_tests/geometries/test_nurbs_curve.cpp

```diff
@@ -14,6 +14,7 @@
 #include <limits>
 
 // External includes
+#include <iomanip>
```

Comment:
Di you need the include?

Reply:
thanks. it is deprecated, I forgot to remove it. it is gone